tiledb/sm/buffer/buffer_list.cc

```diff
@@ -114,6 +121,25 @@ Status BufferList::read(void* dest, uint64_t nbytes, uint64_t* bytes_read) {
   return Status::Ok();
 }
 
+Status BufferList::seek(off_t offset, int whence) {
+  switch (whence) {
+    case SEEK_SET:
+      // We just reset the offsets to 0/start, then fall through to seek_current
+      reset_offset();
+      // fall through
+    case SEEK_CUR:
+      return read(nullptr, offset);
+    case SEEK_END: {
```

Comment:
Does this case need the curly brackets? I can't immediately see a reason to have them here.